### ISSUE TITLE
Creating language Dutch

### ISSUE THREAD

**Issue by rail3r** [django-helpdesk/django-helpdesk#917]
Ok, i am a bit new to this.

I've had a good run getting django-helpdesk to run on the latest Django versions.
Now that i have finaly made in through i am trying to change the language of english to dutch.

I copied the contents of the /locale/en to /locale/nl which is the code for dutch.
Translated some things in it.. but cannot change my language to dutch

in my settings.py i changed the language_code = 'nl-NL' which according to django manual is the one to go with.
also added local_paths = ['/locale/'] to my settings.py

# Internationalization
# https://docs.djangoproject.com/en/2.2/topics/i18n/

LANGUAGE_CODE = 'nl-NL'

TIME_ZONE = 'Europe/Amsterdam'

USE_I18N = True

USE_L10N = True

USE_TZ = True

# Static files (CSS, JavaScript, Images)
# https://docs.djangoproject.com/en/2.2/howto/static-files/

STATIC_URL = '/static/'
LOGIN_URL = '/login/'
LOCALE_PATHS =['/locale/']

Whenever i go to http://localhost:8000/change_language i get an empty list (so no luck there)

I'd probably do something terrible wrong but i cant find out what,..

Thanks for helping in advance!

**Comment by Benbb96** [comment]
You don't need to copy paste the contents of the `/locale/en`. You should run the command makemessages (cf. [docs](https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#message-files)) like so:
`django-admin makemessages -l nl`

Once you have finished edited the created file in `/locale/nl`, you should run the command to compile the messages (cf. [docs](https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#compiling-message-files)) like so : 
`django-admin compilemessages -l nl`

Then the translation should work, if your project settings uses the good language code as you seem to have done correctly 😉

**Comment by gwasser** [comment]
@Benbb96 has the right instructions, please give that a try. If you get a good set of translations, please feel free to open a pull request and contribute them back if you like!

**Comment by Benbb96** [comment]
@rail3r Also note you have to download [gettext ](https://mlocati.github.io/articles/gettext-iconv-windows.html) if you are on Windows in order to run `compilemessages`.

@gwasser Maybe it could be interesting to set up a translate platform tool (like [Transifex](https://www.transifex.com/)) in order to make this process easier for other languages ?

**Comment by gwasser** [comment]
Yep, already have a Transifex account: http://www.transifex.net/projects/p/django-helpdesk/resource/core/

Please feel free to use if that's easier for you!

**Comment by rail3r** [comment]
> @rail3r Also note you have to download [gettext ](https://mlocati.github.io/articles/gettext-iconv-windows.html) if you are on Windows in order to run `compilemessages`.
> 
Thats true for Kubuntu 20.04 also :) installing it now. (even for makemessages)

@gwasser - once im done with the translations, ofcourse i will share them back to the community.

**Comment by rail3r** [comment]
@gwasser, was trying to get myself into transifex but you will have to allow me into the dutch team :)

**Comment by gwasser** [comment]
Oh yes I forgot I needed to do that. I can add you once you have a Transifex account. Please go to the website and login to create your account; you can even hit the GitHub button to use your GitHub credentials to automatically login! Let me know your Transifex username if it's different from rail3r.

**Comment by rail3r** [comment]
Already requested to be part of the team :) rail3r is indeed correct

**Comment by gwasser** [comment]
Added @rail3r! You should be able to translate now. Thank you for your help!

**Comment by rail3r** [comment]
Im stupid in dutch vertalen (translate) looks a lot like verlaten (leave) i pushed the wrong butten :)

**Comment by gwasser** [comment]
No worries, I added you back @rail3r

**Comment by uhurusurfa** [comment]
Closing as this seems tio have been resolved and is quite old now.
Please open a new issue if you still have issues.
